lib/galaxy/webapps/tool_shed/controllers/repository.py

```diff
@@ -1713,8 +1713,8 @@ def get_tool_dependencies( self, trans, **kwd ):
         owner = kwd.get( 'owner', None )
         changeset_revision = kwd.get( 'changeset_revision', None )
         repository = suc.get_repository_by_name_and_owner( trans.app, name, owner )
```

Comment:
That was expected: https://github.com/galaxyproject/galaxy/pull/2950#discussion_r80083956